lib/logger.rb

```diff
@@ -234,7 +241,34 @@ module Severity
   include Severity
 
   # Logging severity threshold (e.g. <tt>Logger::INFO</tt>).
-  attr_accessor :level
+  attr_reader :level
+
+  # Set logging severity threshold.
+  #
+  # +severity+:: The Severity of the log message.
+  def level=(severity)
+    if severity.is_a?(Integer)
+      @level = severity
+    else
+      _severity = severity.to_s.downcase
+      case _severity
+      when 'debug'.freeze
```

Comment:
logger already has several `.freeze` codes such as https://github.com/ruby/ruby/blob/508addd7e06fac19fe36fee957b31035e72097db/lib/logger.rb#L211, https://github.com/ruby/ruby/blob/508addd7e06fac19fe36fee957b31035e72097db/lib/logger.rb#L484, https://github.com/ruby/ruby/blob/508addd7e06fac19fe36fee957b31035e72097db/lib/logger.rb#L497, https://github.com/ruby/ruby/blob/508addd7e06fac19fe36fee957b31035e72097db/lib/logger.rb#L513. I want to keep consistency.